docs/modules/operation/pages/deep-dive/alarms/alarm-handling.adoc

```diff
@@ -0,0 +1,52 @@
+
+= Alarm handling
+
+The following are ways you can interact with alarms.
+
+== Acknowledgment
+
+PICTURES MISSING
+
+Users can acknowledge alarms.
+The alarm will be moved out from `Alarm(s) outstanding` into the `Alarm(s) acknowledged` view.
+Doing this lets other {page-component-title} users see that someone is aware of the alarm.
+
+NOTE: A manually acknowledged alarm that was cleared automatically by its clear event needs also unacknowledged manually to disappear.
+
+== Clearing
+
+PICTURES MISSING
+
+Clearing an outstanding or acknowledged alarm means "mark it as resolved".
+The cleared alarm will disappear within a defined cycle.
+
+NOTE: Alarms created by a threshold configuration will appear again if the defined metric still matches the threshold configurations or matches it again.
+An alarm based on a `nodeLostService` event won't appear again.
+To keep track of those `outage alarms`, check the outages list.
```

Comment:
Any new event with alarm data will re-open a cleared alarm. Technically a new nodeLostService event will re-open an existing cleared alarm.  It's just that the system won't generate another copy of that event unless the service comes back online.
```suggestion
```

Reply:
Maybe the difference here is, how the alarm was cleared or if there is still an outage entry for it. When I clear a `nodeLostService` alarm manually, it gets removed after 5 mins. I still have an outage for this service. Then I will never get a new alarm for this one anymore. Also a restart of ONMS won't trigger this.
But this is different for threshold alarms. If new collection samples get checked and the threshold fires, I will get an alarm again.

This is what I wanted to tell here.

Reply:
it is more a matter if the event will be re-sent.  Once a service is down, there won't be another down event generated unless the event comes back up.  Threshold events can be re-sent if the threshold is detected as breached again.